controllers/datadogagent/controller_test.go

```diff
@@ -1167,6 +1167,14 @@ func TestReconcileDatadogAgent_Reconcile(t *testing.T) {
 				if dca.OwnerReferences == nil || len(dca.OwnerReferences) != 1 {
 					return fmt.Errorf("dca bad owner references, should be: '[Kind DatadogAgent - Name foo]', current: %v", dca.OwnerReferences)
 				}
+
+				clusterAgentContainer := dca.Spec.Template.Spec.Containers[0]
```

Comment:
this test file is for the v1alpha1 CRD (deprecated and will remove the code soon)

Reply:
Removed the test 👍🏻